dipy/align/tests/test_imaffine.py

```diff
@@ -117,28 +117,28 @@ def test_transform_geometric_centers_3d():
                     moving_grid2world = npl.inv(static_grid2world)
 
                     # Expected translation
-                    c_static = np.array(shape_static, dtype = np.float64) * 0.5
+                    c_static = np.array(shape_static, dtype=np.float64) * 0.5
                     c_static = tuple(c_static)
                     c_static = static_grid2world.dot(c_static+(1,))[:3]
-                    c_moving = np.array(shape_moving, dtype = np.float64) * 0.5
+                    c_moving = np.array(shape_moving, dtype=np.float64) * 0.5
                     c_moving = tuple(c_moving)
                     c_moving = moving_grid2world.dot(c_moving+(1,))[:3]
-                    expected = np.eye(4);
+                    expected = np.eye(4)
                     expected[:3, 3] = c_moving - c_static
 
                     # Implementation under test
-                    actual = imaffine.transform_geometric_centers(static,
-                        static_grid2world, moving, moving_grid2world)
+                    actual = imaffine.transform_geometric_centers
```

Comment:
Here, function `imaffine.transform_geometric_centers` is not being called. Instead, `actual` will be assigned a reference to the function, that's why Travis failing. To call the function, at least the opening parenthesis `(` must appear before breaking the line.